benefits/admin.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+
+
+STAFF_GROUP_NAME = "Cal-ITP"
```

Comment:
Is it possible to reuse the existing `STAFF_GROUP_NAME` variable from `benefits.core.admin`?

(I'm not sure if maybe this is just here temporarily while you focus on the bigger task of implementing the templates. But in any case, let's avoid duplicating this string.)

Reply:
Yep, it's here temporarily to make the local debugging setup easier (changing from `Cal-ITP` to `CST`, for example) but for sure I'll reuse the existing `STAFF_GROUP_NAME` 👍

Reply:
Would it be possible to make this a setting and use it that way everywhere? I think that would be cleaner than a Python variable import.

Reply:
Yep, I made it a setting in ee80de36ed8c2ef456c7fba44a01fa6f2cb1a4a6. Then, I made a small update to [`core/admin.py`](https://github.com/cal-itp/benefits/blob/main/benefits/core/admin.py) and its test in 62a2e8ff0bceb409cffbe1513dca74f01ce81a07 as part of this PR.